Formula/darkice.rb

```diff
@@ -21,6 +22,9 @@ class Darkice < Formula
   depends_on "jack"
 
   def install
+    ["aacPlusEncoder.cpp", "FaacEncoder.cpp", "OpusLibEncoder.cpp", "VorbisLibEncoder.cpp"].each do |f|
```

Comment:
Cool. Please add the link, date, and brief description like these: https://gist.github.com/ilovezfs/62a9eb715dc044f7202cbf86caaf713b